<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectExplorer::Internal::ProjectExplorerSettingsPageUi</class>
 <widget class="QWidget" name="ProjectExplorer::Internal::ProjectExplorerSettingsPageUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>808</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="directoryGroupBox">
     <property name="title">
      <string>Projects Directory</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="currentDirectoryRadioButton">
        <property name="text">
         <string>Current directory</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">directoryButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="directoryRadioButton">
        <property name="text">
         <string>Directory</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">directoryButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Utils::PathChooser" name="projectsDirectoryPathChooser"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="closeProjectGroupBox">
     <property name="title">
      <string>Closing Projects</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="closeSourceFilesCheckBox">
        <property name="text">
         <string>Close source files along with project</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="buildAndRunGroupBox">
     <property name="title">
      <string>Build and Run</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="0">
       <widget class="QCheckBox" name="abortBuildAllOnErrorCheckBox">
        <property name="text">
         <string>Abort on error when building all projects</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="jomCheckbox">
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Use jom instead of nmake</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="jomLabel">
          <property name="text">
           <string>&lt;i&gt;jom&lt;/i&gt; is a drop-in replacement for &lt;i&gt;nmake&lt;/i&gt; which distributes the compilation process to multiple CPU cores. The latest binary is available at &lt;a href=&quot;http://download.qt.io/official_releases/jom/&quot;&gt;http://download.qt.io/official_releases/jom/&lt;/a&gt;. Disable it if you experience problems with your builds.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="indent">
           <number>20</number>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QWidget" name="widget_1" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="lowBuildPriorityCheckBox">
        <property name="toolTip">
         <string>Enable this if your system becomes unresponsive while building.</string>
        </property>
        <property name="text">
         <string>Start build processes with low priority</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="deployProjectBeforeRunCheckBox">
        <property name="text">
         <string>Always deploy project before running it</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="addLibraryPathsToRunEnvCheckBox">
        <property name="text">
         <string>Add linker library search paths to run environment</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="saveAllFilesCheckBox">
        <property name="text">
         <string>Save all files before build</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="automaticallyCreateRunConfiguration">
        <property name="toolTip">
         <string>Creates suitable run configurations automatically when setting up a new kit.</string>
        </property>
        <property name="text">
         <string>Create suitable run configurations automatically</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="promptToStopRunControlCheckBox">
        <property name="toolTip">
         <string>Asks before terminating the running application in response to clicking the stop button in Application Output.</string>
        </property>
        <property name="text">
         <string>Always ask before stopping applications</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="clearIssuesCheckBox">
        <property name="text">
         <string>Clear issues list on new build</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="buildBeforeDeployLabel">
          <property name="text">
           <string>Build before deploying:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="buildBeforeDeployComboBox"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelStopBeforeBuild">
          <property name="text">
           <string>Stop applications before building:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="stopBeforeBuildComboBox"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="terminalModeLabel">
          <property name="text">
           <string>Default for &quot;Run in terminal&quot;:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="terminalModeComboBox">
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Deduced from Project</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QLineEdit</extends>
   <header>utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="directoryButtonGroup"/>
 </buttongroups>
</ui>
